packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -17,14 +20,31 @@ import 'memory_service.dart';
 
 typedef chartStateListener = void Function();
 
+const String _filenamePrefix = 'memory_log_';
+
+// Memory Log filename.
+final String _memoryLogFilename =
```

Comment:
make this a method so that it can be generated. As is, the name will only be generated on creation, which would be incorrect for subsequent log files.

Reply:
I only generate one log file, for now, it keeps updating the last log since start of the devtools.  I'll add a todo to add support for more than one.